pages/index.vue

```diff
@@ -1,3 +1,76 @@
 <template>
-    <h1>Hello world!</h1>
-</template>
\ No newline at end of file
+    <div class="my-team">
+        <header class="my-team__main-title">
+            <h1>The Creative crew</h1>
+        </header>
+        <div class="my-team__description">
+            <h2>Who we are</h2>
+            <p>
+                We are a team of creatively diverse. driven. innovative individuals
+                working in various locations from the world
+            </p>
+        </div>
+        <div class="my-team__members">
+            <article class="my-team__members__member-card" v-for="member of members">
+                <img class="my-team__members__member-card__photo" v-bind:src="member.photoUrl" alt=""/>
+                <div class="my-team__members__member-card__job-title">{{ member.jobTitle }}</div>
+                <h2 class="my-team__members__member-card__name">{{ member.name }}</h2>
+            </article>
```

Comment:
We should try to avoid long chain of BEM convention.

You could treat the member card as a new element to reset the BEM.

```
<div class="my-team__members">
  <article class="member-card" v-for="member of members">
    <img class="member-card__photo" v-bind:src="member.photoUrl" alt=""/>
    <div class="member-card__job-title">{{ member.jobTitle }}</div>
    <h2 class="member-card__name">{{ member.name }}</h2>
  </article>
</div>
```

Reply:
Can we move this to a component?

Reply:
Yes, I'll move  it